src/Common/MoistThermodynamics/states.jl

```diff
@@ -267,7 +289,7 @@ Constructs a [`PhaseNonEquil`](@ref) thermodynamic state from:
  - `ρ` (moist-)air density
  - `q_pt` phase partition
 and, optionally
- - `tol` tolerance for non-linear equation solve
+ - `temperature_tol` estimated temperature tolerance for saturation adjustment
```

Comment:
I thought this is non-equilibrium, so why do we need any saturation adjustment? (If we do, as above, the tolerance should be a potential temperature tolerance.)

Reply:
This non-linear equation shows up when we need to construct a non-equilibrium phase with `θ_liq_ice, ρ, q_pt` (because we must somehow compte temperature from these arguments). We replaced the particular equation docs with
`T - air_temperature_from_liquid_ice_pottemp_given_pressure(param_set, θ_liq_ice, air_pressure(param_set, T, ρ, q), q) = 0`
This should only ever be used for initialization when [`LiquidIcePotTempSHumNonEquil`](https://github.com/CliMA/ClimateMachine.jl/blob/master/src/Common/MoistThermodynamics/states.jl#L273) constructor is used. Also, it's used as a [comparison against our approximation in `air_temperature_from_liquid_ice_pottemp`](https://github.com/CliMA/ClimateMachine.jl/blob/master/test/Common/MoistThermodynamics/runtests.jl#L615).

Reply:
Ok. Saturation adjustment is a misnomer in this case then. We do not adjust the saturation state (it is given by `q`). It is just a nonlinear solve for temperature. It would be good to make this clearer.